src/main/java/newbank/server/exceptions/AccountInvalidNameException.java

```diff
@@ -0,0 +1,7 @@
+package newbank.server.exceptions;
+
+public class AccountInvalidNameException extends Exception {
```

Comment:
Did you mean "InvalidAccountNameException"?

Kindest Regards,
Jules

Reply:
I wanted to prefix using `Account`, and then the error type (in natural language): `InvalidName`. But it is kind of true `AccountNameInvalid` is more consistent here.

Reply:
Indeed. "AccountNameInvalid" reads better.

Kindest Regards,
Jules

Reply:
Hi Jules,

I've had a second thought on this and `AccountInvalidNameException` sounds better to me, as it's a combination of the prefix `Account` (easily searchable) and the actual error name `InvalidNameException`.

`AccountNameInvalidException` sounds as if it would be an `InvalidException` on an `AccountName`, which is somewhat correct, but not my original intention (we don't have a class `AccountName`).

Could we accept it as it is now and create a ticket to consider how we plan to organize our exceptions?

Best,
Arek

Reply:
AccountNameInvalidException  sounds good.